Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "crc"
-version = "1.2.0"
+version = "1.3.0"
```

Comment:
we usually use separate commit to bump and tag version
so only keep the bump of lazy_static in this pr is good enough